rag-langchain-python/vector_query_retriever.py

```diff
@@ -34,6 +34,7 @@ def _build_eval_script(self, query, query_embedding):
     null, {{'scoreMethod': 'score-bm25'}}
   )
   .limit(100)
+  .bind(op.as(op.col('transcript'), op.xpath(op.col('doc'), '/transcript')))
```

Comment:
I think you can simplify this to:

    .bind(op.as("transcript", op.xpath("doc", "/transcript")))

But not 100% certain. I think the fewer times `op.` is used in an Optic plan, the better, as it's just noise most of the time.